.release-it.json

```diff
@@ -0,0 +1,25 @@
+{
+    "git": {
+      "requireCleanWorkingDir": true,
+      "commitMessage": "Release v${version}",
+      "tagAnnotation": "Release v${version}",
+      "tagName": "${version}",
+      "push": true
+    },
+    "github": {
```

Comment:
`title`, `body` and `base` should be underneath a `pr` property: https://github.com/Amsterdam/ee-docs/blob/develop/.release-it.json#L12